packages/parser/src/components/parser/statement/expressionStatement.ts

```diff
@@ -0,0 +1,22 @@
+import Statement from '.';
+
+import { TokenTypes } from '../../../constants/bhaiLangSpec';
+
+import Expression from './expression';
+import { ExpressionType } from './expression/constants';
+
+
+export default class ExpressionStatement extends Statement {
+
+
+  getStatement() {
+    const expression = Expression.getExpressionImpl(ExpressionType.AdditiveExpression); 
```

Comment:
Same thing as above: This will return expression Impl not thee actual expression. I think we need to call getExpression().

Reply:
Lets make sure that at every place we call getExpression() whereever we have getExpressionImpl